virtcontainers/pkg/firecracker/firecracker.yaml

```diff
@@ -5,7 +5,7 @@ info:
                The API is accessible through HTTP calls on specific URLs
                carrying JSON modeled data.
                The transport medium is a Unix Domain Socket.
-  version: 0.19.0
+  version: 0.21.0
```

Comment:
I'm assuming this is version number is correct (pristine upstream file where presumably they just ignore the patch level here) even though it doesn't match the full version being supported by this PR.

Reply:
updated~~thanks. ;)